### PR TITLE
EID-634: Update release notes for MSA release 3.0.0

### PR DESCRIPTION
**Pull request by andy-paine**

### CODE REVIEW

File: doc/release-details/3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+To enable support for European identities, add the `europeanIdentity` section below to your [MSA configuration file](http://alphagov.github.io/rp-onboarding-tech-docs/pages/matching/matchingserviceadapter.html#in-the-field-europeanidentity).
+
+```yaml
+europeanIdentity:
+  enabled: ${EUROPEAN_IDENTITY_ENABLED} # true or false
+  hubConnectorEntityId: https://www.integration.signin.service.gov.uk/SAML2/metadata/connector # The URL of the metadata for the node that requests and receives identities from European countries.
+  # Configure metadata for European countries.
+  aggregatedMetadata:
+    trustAnchorUri: https://www.integration.signin.service.gov.uk/SAML2/metadata/trust-anchor # The location of the trust anchor used to validate country metadata
```
Comment (IreneLau-GDS) [inline doc/release-details/3.0.0.md]: Need the ```jerseyClientName: trust-anchor-client``` here.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,9 +2,27 @@ MSA Release Notes
...
+ hub:
+   ssoUrl: https://integration.ida.digital.cabinet-office.gov.uk/SAML2/SSO
+-  trustStore:
+-    path: test_ida_hub.ts
+-    password: puppet
+
+ metadata:
++  # Choices are (INTEGRATION|PRODUCTION)
++  environment: INTEGRATION
+```
```
Comment (IreneLau-GDS) [inline RELEASE_NOTES.md]: Do we need to mention anything about the eIDAS config here?

Comment (andy-paine) [reply RELEASE_NOTES.md]: I don't think so - these are _required_ config changes - your MSA will break unless you make these. You can freely leave out the whole eIDAS section and everything will be fine so it doesn't seem compulsory.

Comment (bravegrape) [reply RELEASE_NOTES.md]: That's in the release details for accepting European identities. 
It says what to add to the config file to be able to consume European identities.

**Pull request merged**